src/server/routers/_app.ts

```diff
@@ -242,6 +242,21 @@ export const appRouter = router({
     .mutation(async ({ ctx, input }) => {
       const res = await axios.post(process.env.APPLICATION_URL as string, input)
     }),
+
+  acceptProposalApplication: publicProcedure
+    .input(
+      z.object({
+        proposalId: z.string(),
+        proposalApplicationId: z.string(),
+        applicantEmail: z.string().email(),
+      })
+    )
+    .mutation(async ({ ctx, input }) => {
+      const res = await axios.post(
+        process.env.APPLICATION_ACCEPTANCE_URL as string,
+        input
+      )
```

Comment:
return the response from the flow -> if the flow suceeds, we can send a response to the backend, and the backend can return it to the frontend. on success, the frontend can refetch data. for simple flows like this one, this can allow automated data refresh on the frontend without hacky local storage solutions. only the flows that take a long time could not be done like that

Reply:
at least I believe it would work like that. maybe the flow would need an explicit "response" action or something, to actually return an http response

Reply:
I think my solution works just fine. Don't see any reason to change it. 
If you have a SPECIFIC suggestion - push your change to this branch.